src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx

```diff
@@ -1,11 +1,13 @@
 import React from "react"
+import moment from "moment"
```

Comment:
Lump moment in with underscore as forbidden dependencies in new code and please call out their usage if you see this in code review. You can replace this with luxon.